File: oeml-sdk/dart-jaguar.yaml/lib/model/order_cancel_all_request.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'order_cancel_all_request.jser.dart';

class OrderCancelAllRequest {
   /* Identifier of the exchange from which active orders should be canceled. */
  @Alias('exchange_id', isNullable: false,  )
  final String exchangeId;
  

  OrderCancelAllRequest(
      

{
    
     this.exchangeId = null 
    }
  );

  @override
  String toString() {
    return 'OrderCancelAllRequest[exchangeId=$exchangeId, ]';
  }
}

@GenSerializer(nullableFields: true)
class OrderCancelAllRequestSerializer extends Serializer<OrderCancelAllRequest> with _$OrderCancelAllRequestSerializer {

}

File: chain-sdk/javascript-closure-angular/API/Client/OrderCancelAllRequest.js
goog.provide('API.Client.OrderCancelAllRequest');

/**
 * Cancel all orders request object.
 * @record
 */
API.Client.OrderCancelAllRequest = function() {}

/**
 * Identifier of the exchange from which active orders should be canceled.
 * @type {!string}
 * @export
 */
API.Client.OrderCancelAllRequest.prototype.exchangeId;

